packages/react/src/Footer/MenuIcons.client.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from "react";
+import { DynamicIcon } from "./DynamicIcons.client";
+
+export type SocialMedia = {
+  icon: string;
```

Comment:
Pourquoi ne pas mettre un UnionType ?
```suggestion
  icon: 'facebook' | 'insta' |...;
```